src/main/java/com/epam/ta/reportportal/core/ElementsCounterService.java

```diff
@@ -31,9 +32,11 @@
 @Service
 public class ElementsCounterService {
 
-	private TestItemRepository testItemRepository;
+	private static final int BATCH_SIZE = 50;
```

Comment:
Could you move it to variables please, could be needed for more convenient for perf testing

Reply:
done